charmhub_lp_tools/main.py

```diff
@@ -74,14 +68,18 @@
     CharmProject,
     setup_logging as cp_setup_logging,
 )
-
 from .charmhub import setup_logging as ch_setup_logging
+from .reports import (
+    get_builds_report_klass,
+    get_supported_report_types,
+)
 
 
 logger = logging.getLogger(__name__)
 
 LOGGING_FORMAT = "%(asctime)s %(levelname)s %(name)s %(message)s"
 NOW = datetime.now(tz=ZoneInfo("UTC"))
+__THIS__ = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I think this may be a cut-and-paste leftover?  e.g. it's also in the `./reports.py` file.

Reply:
ack